tasks/js-basics.md

```diff
@@ -55,8 +55,10 @@ When you finish this task you can proceed to the next one.
 ## Extra materials
 
 - [Курс "Введение в программирование" от Hexlet](https://ru.hexlet.io/courses/introduction_to_programming)
+- [FunFunFunction - Functional programming in JavaScript](https://www.youtube.com/playlist?list=PL0zVEGEvSaeEd9hlmCXrk5yUyqUag-n84)
 - [Algorithms Course by Khan Academy](https://www.khanacademy.org/computing/computer-science/algorithms)
 - [Visualization of Common Algorithms and Data Structures](https://www.cs.usfca.edu/~galles/visualization/Algorithms.html)
+- [JavaScript Algorithms and Data Structures examples](https://github.com/trekhleb/javascript-algorithms)
```

Comment:
I think for newbies will be quite complicated and confused for understanding

Reply:
This is extra material and optional for further reading. It contains many examples, and someone will be interested to see how to write common algorithms with java script, I guess.